src/main/plugin/iso19139.ca.HNAP/templates/vector-unilingual.xml

```diff
@@ -82,6 +82,22 @@
   <gmd:dataSetURI gco:nilReason="missing">
     <gco:CharacterString/>
   </gmd:dataSetURI>
+  <gmd:locale>
+    <gmd:PT_Locale id="eng">
+      <gmd:languageCode>
+        <gmd:LanguageCode codeList="http://nap.geogratis.gc.ca/metadata/register/napMetadataRegister.xml#IC_116"
+                          codeListValue="eng">English; Anglais</gmd:LanguageCode>
```

Comment:
Do you find it odd that a uni-lingual record has bilingual code values?
This may be a hard one to fix!

Reply:
yes - definitely weird.  I think we need to verify we are using the correct code list....

This might be ok -- however, not 100% sure about a uni-lingual HNAP (or would that just be NAP)?